packages/editor/src/components/autosave-monitor/index.js

```diff
@@ -11,18 +11,18 @@ import { store as coreStore } from '@wordpress/core-data';
  */
 import { store as editorStore } from '../../store';
 
-/**
- * AutosaveMonitor invokes `props.autosave()` within at most `interval` seconds after an unsaved change is detected.
- *
- * The logic is straightforward: a check is performed every `props.interval` seconds. If any changes are detected, `props.autosave()` is called.
- * The time between the change and the autosave varies but is no larger than `props.interval` seconds. Refer to the code below for more details, such as
- * the specific way of detecting changes.
- *
- * There are two caveats:
- * * If `props.isAutosaveable` happens to be false at a time of checking for changes, the check is retried every second.
- * * The timer may be disabled by setting `props.disableIntervalChecks` to `true`. In that mode, any change will immediately trigger `props.autosave()`.
- */
 export class AutosaveMonitor extends Component {
+	/**
```

Comment:
I think it's fine to keep these comments just once, below. No strong opinion though..

Reply:
If I remove this and fix the JSDoc issue then nothing is generated in README and just shows `Undocumented declaration.` when I run `npm run docs:build`. This is some odd stuff. I think I picked the wrong item to start with. haha!

Reply:
🤔 This isn't the case for me. These lines are ignored by the docs generation script..

It's just the first PR's to get things rolling! Maybe it's better to have to solve these issues from the start. After that everything will be super easy 😄